lib/human_player.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'player'
+require_relative 'user_interface'
+require_relative 'validation'
+
+class HumanPlayer < Player
+  include Validation
+  include UserInterface
+  attr_accessor :output, :input
+
+  def initialize(name, symbol)
+    super(name, symbol)
+    @output = $stdout
+    @input = $stdin
```

Comment:
I would think that we can hide output and input behind UserInterface, so that HumanPlayer doesn't need to know about them.

Reply:
Resolved at ff95fbb6e155906d4c2cd1434215bdaa182646ac